transports/webrtc/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "libp2p-webrtc"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "WebRTC transport for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+edition = "2021"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+asynchronous-codec = "0.6"
+async-trait = "0.1"
+bytes = "1"
+fnv = "1.0"
+futures = "0.3"
+futures-lite = "1"
+futures-timer = "3"
+hex = "0.4"
+if-watch = "2.0"
+libp2p-core = { version = "0.37.0", path = "../../core", default-features = false  }
```

```suggestion
libp2p-core = { version = "0.37.0", path = "../../core"  }
```

This is no longer necessary.